Chrom Soaring Aether

CSharpM7 opened the pull request:
⚠️WIP. Balancing/Code Cleanup soon (tm)⚠️
https://github.com/HDR-Development/HewDraw-Remix/assets/13909643/0652e4a3-41aa-495a-907f-b73d9b792ea0

Required assets:
[assets-chrom.zip](https://github.com/HDR-Development/HewDraw-Remix/files/15049190/assets-chrom.zip)


Changes Soaring Slash into a multi-part Up Special. Chrom starts by slashing his sword and rising upwards. If the special button is released, Chrom will flip upwards and enter special fall, allowing him to grab the ledge earlier than his original recovery. During the flip-out, Chrom has a decent amount of control over where he can drift. If the special button is held or pressed early into the flip, Chrom will enter Aether, and plummet downwards while spinning. Opponents can get caught in this whirlwind, and their only escape might be to SDI out of the move. If either 30 frames pass, or the player releases the special button, Chrom will perform a launching attack during his plummet. If the player knows the opponent can SDI well, or is in a poor position to maximize damage, they are encouraged to release the special button early to cash out on the reward this move gives them.

CSharpM7 committed:
chrom upB

CSharpM7 committed:
kinetic / damage update

CSharpM7 committed:
Update special_hi.rs

CSharpM7 commented:
This has ruined my [🚫](https://emojipedia.org/prohibited) credibility on twitter dot com

CSharpM7 committed:
Update specials.rs

CSharpM7 committed:
Cancelable flip out

CSharpM7 committed:
Update special_hi.rs

zrksyd commented:
> This has ruined my [🚫](https://emojipedia.org/prohibited) credibility on twitter dot com

[Putting this here for a video reference.](https://x.com/csharpm7/status/1768857669367111818?s=46&t=Fo3Ruukla3D1DnK7zimwXQ)

CSharpM7 committed:
Cancellable on hit only

SkewedAskew committed:
Merge branch 'dev' into chrom-upB

SkewedAskew committed:
Morch

- Removed actionability
- Dive transition changed to pressing Special within a certain window of the flip
- Split "dive start" into its own animation
- Refactored acmd and status code
- Adjusted uncharged neutral special vfx

SkewedAskew committed:
Update specials.rs

SkewedAskew committed:
Bugfixes
